src/auto.ts

```diff
@@ -308,14 +308,18 @@ export default class Auto {
     this.logger.verbose.info('Posting comment to GitHub\n', msg);
 
     if (!dryRun) {
-      await this.git.createStatus({
-        ...options,
-        ...msg,
-        target_url,
-        sha
-      } as IPRInfo);
-
-      this.logger.log.success('Posted status to Pull Request.');
+      try {
+        await this.git.createStatus({
+          ...options,
+          ...msg,
+          target_url,
+          sha
+        } as IPRInfo);
+
+        this.logger.log.success('Posted status to Pull Request.');
+      } catch (error) {
+        this.logger.verbose.error('Failed to post status to github');
```

Comment:
The status creation call here is what actually ended up leaking the token. It resulted in a 404. This message probably should be wordsmithed a little. Should the error always print out, or only print in verbose mode?

Reply:
I think it makes more sense to include the error in the default log. After that this PR looks good to go

Reply:
```suggestion
        this.logger.log.error('Failed to post status to github');
```